src/System.Diagnostics.Process/tests/ProcessTests.Linux.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Security;
+using Xunit;
+using Xunit.NetCore.Extensions;
+
+namespace System.Diagnostics.Tests
+{
+    public partial class ProcessTests : ProcessTestBase
+    {
+        [Fact]
+        [OuterLoop("Launches default application")]
+        public void NewXdg_TestWithFilename_ShouldUseOpenWithDefaultApp()
+        {
+            string fileToOpen = Path.Combine(Environment.CurrentDirectory, "..", "..", "..", "..", "PATENTS.TXT");
```

Comment:
Better to have a txt file in the tests\ directory and have this in your test csproj
```xml
        <Content Include="foo.txt">
          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
        </Content>
```